nservicebus/upgrades/5to6.md

```diff
@@ -442,12 +442,14 @@ snippet: 5to6CriticalError
 
 ## Notifications
 
-The `BusNotifications` class previously exposed the available notification hooks as observables implementing `IObservable`. This meant a custom `IObserver` and the use of [Reactive-Extensions](https://msdn.microsoft.com/en-au/data/gg577609.aspx) was required to use this API. 
+The `BusNotifications` class has been renamed to `Notifications`. 
 
-In Versions 6 and above the notifications API has been changed `BusNotifications` class has been renamed to `Notifications`. It now also exposes regular events instead of observables for easier usage. Find out more about [subscribing to error notifications](/nservicebus/errors/subscribing-to-error-notifications.md). To continue using Reactive-Extensions the events API can be transformed into `IObservable`s like this:
+`BusNotifications` previously exposed the available notification hooks as observables implementing `IObservable`. This meant a custom `IObserver` and the use of [Reactive-Extensions](https://msdn.microsoft.com/en-au/data/gg577609.aspx) was required to use this API. With Version 6 the notifications API has been changed to expose regular events instead of observables for easier usage. To continue using Reactive-Extensions the events API can be transformed into `IObservable`s like this:
```

Comment:
> This meant a custom `IObserver` and the use of [Reactive-Extensions](https://msdn.microsoft.com/en-au/data/gg577609.aspx) was required to use this API.

Simplify: This required using `IObserver` and Reactive-Extensions](https://msdn.microsoft.com/en-au/data/gg577609.aspx).

> With Version 6

In Version 6...

> With Version 6 the notifications API has been changed to expose regular events instead of observables for easier usage.

In Version 6 the notifications API has been changed for easier usage. It exposes regular events instead of observables.

> To continue using Reactive-Extensions the events

Shouldn't there be a comma after "Reactive-Extensions"?